examples/deployment-manager/autogen/singlevm/README.md

```diff
@@ -0,0 +1,64 @@
+## Prerequisites
+
+This example package generates a deployment manager template for a wordpress GCE
+VM. Your GCP project must contain a wordpress VM image to use in the deployment
+manager template. The following command copies the wordpress VM image from the
+[click-to-deploy project](https://github.com/GoogleCloudPlatform/click-to-deploy)
+to your GCP project.
+
+```
+PROJECT_ID=<YOUR_PROJECT_ID>
+gcloud compute --project=$PROJECT_ID images create wordpress --source-image=wordpress-v20200629 --source-image-project=click-to-deploy-images
+```
+
+## Generating Deployment Manager Template
+
+To use this mpdev package, execute the following command which will checkout this
```

Comment:
`kpt` and `mpdev` generic instruction should probably reside in another general guide. For these package specific READMEs, perhaps we should only include the commands and reference the general guide instead.